install/linux/usr/share/odemis/sim/sparc2-time-correlator-sim.odm.yaml

```diff
@@ -421,8 +412,9 @@ SPARC2-VisNIR-TR: {
     init: {
         axis_name: "od",
         # position when referenced (adjusted to be a multiple of the actuator stepsize)
-        offset: -3.00000925, # od
-        ref_start: 2.8 #  a little before the reference (inverted axis)
+        offset: -2.98, # od
+        ref_start: 2.78, #  a little before the reference (inverted axis)
+        ref_period: null,  # never automatically reference the axis
```

Comment:
Could you update the comment with the unit of the reference period when the auto reference is needed?